### DISCUSSION TITLE
Release of Microsoft.ML.OnnxRuntime.QNN nuget with Qualcomm SDK dependencies

### DISCUSSION THREAD

**Discussion by rgsousa88** [microsoft/onnxruntime#19625] [EP Q&A] [answered]
Hi,
As described in Microsoft.ML.OnnxRuntime.QNN nuget [page](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.QNN), to use this nuget to perform .onnx model inference in HTP devices is required that Qualcomm SDK be installed. Do you have any plan to release a version of this nuget including .dll from Qualcomm SDK in such way that install this SDK wouldn't be necessary?

**Comment by jeffmend** [comment, marked as answer]
No, we do not have a plan to redistribute Qualcomm's SDK as part of our package.
